### PR TITLE
Minor change in symfony guide

### PR DESCRIPTION
**Pull request by technodelight**
Suggest users to create a docker hub account and link in documentation.

### CODE REVIEW

File: content/guides/symfony.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ When creating the `continuous-pipe.yml` above an `IMAGE_NAME` and `CLUSTER` vari
 
 The `CLUSTER` value you need can be found in the "Clusters" tab of the team. If you manually entered the Kubernetes cluster details you will have set the value yourself. If you used a Google linked account the value will have been set when setting up the Google Container Engine.
 
-The `IMAGE_NAME` value is the path where the Docker image should be pushed to. This needs to be the full name, including the Docker repo and account e.g. `docker.io/continuouspipe/demo-symfony`.
+The `IMAGE_NAME` value is the path where the Docker image should be pushed to. This needs to be the full name, including the Docker repo and account e.g. `docker.io/continuouspipe/demo-symfony`. If you don't yet have a Docker repo nor you know how to create one, please refer to [Docker Hub Repositories documentation](https://docs.docker.com/docker-hub/repos/).
```
Comment (pswaine) [inline content/guides/symfony.md]: Good idea 👍 I'd suggest rewording it to:
"If you don't yet have a Docker repo nor **do** you know how to create one, please refer to **the** [Docker Hub **r**epositories documentation]"

Comment (technodelight) [inline content/guides/symfony.md]: 👍

**Comment by technodelight** [comment]
@pswaine I've updated the wording :) Thanks!
